tests/failpoints/cases/test_disk_full.rs

```diff
@@ -81,7 +94,38 @@ fn test_disk_full_allowed_leader_behaviors() {
     cluster.pd_client.must_add_peer(1, new_peer(3, 3));
     must_get_equal(&cluster.get_engine(3), b"k1", b"v1");
 
-    fail::remove("disk_full_peer_1");
+    // Test Txn commit/rollback is also allowed.
```

Comment:
Seems rollback is not covered. And please extract test code about transaction to another test case. This case only cares raft layrer instead of transaction layer.

Reply:
And, we also need a cases that prewrite reuqests with `DislFullOpt::Allowed` should be able to be proposed.

Reply:
Right, these are already covered, like "Test Txn Prewrite is allowed with AllowedOnAlmostFull flag."